src/osd/scrubber/scrub_backend.cc

```diff
@@ -1171,7 +1171,7 @@ bool ScrubBackend::compare_obj_details(pg_shard_t auth_shard,
 
   if (auth.digest_present && candidate.digest_present &&
       auth.digest != candidate.digest) {
-    format_to(std::back_inserter(out),
+    fmt::format_to(std::back_inserter(out),
```

Comment:
why is this required? we (ceph) should not use std::format until a possible full transition to that version
(which should probably wait for c++23 full availability on both Clang & gcc)

Reply:
because the standard library shipped along with GCC 13 actually implemented `std::format_to()`. so without adding `fmt::`, the compiler would fail to disambiguate the one from libstdc++ and the one from fmtlib.